### PR TITLE
Refix linter rules

### PR DESCRIPTION
**Pull request by devorah**
Looks like some of the linter fixes were overwritten and lost from this original commit: https://github.com/tova-moskowitz/bad_poetry/pull/2/commits/4137532c47935e4d31dc2232bb963eb2a3999072

This PR is to add them back and fix code style/indentation to conform to new linter rules.

### CODE REVIEW

**Comment by devorah** [comment]
@tova-moskowitz - please review.

File: .eslintrc.json [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-      "comma-dangle": 0,
-      "indent": ["error", 2, { "SwitchCase": 2, "VariableDeclarator": 2, "outerIIFEBody": 2 }],
-      "react/jsx-indent-props": ["error", 2],
-      "react/jsx-indent": ["error", 2],
-      "object-shorthand": ["off", "always"],
-      "func-names": ["off", "always"]
-
-  }
+    "rules": {
+        "react/prefer-es6-class": 1,
```
Comment (tova-moskowitz) [inline .eslintrc.json:13]: This is new, yes? I approve! Best way to learn es6

File: .eslintrc.json [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
+        "react/prefer-stateless-function": 0,
+        "object-shorthand": 1,
+        "comma-dangle": 0,
+        "indent": ["error", 4, { "SwitchCase": 1, "VariableDeclarator": 1, "outerIIFEBody": 1 }],
+        "react/jsx-indent-props": ["error", 4],
+        "react/jsx-indent": ["error", 4],
+        "object-shorthand": ["off", "always"],
```
Comment (tova-moskowitz) [inline .eslintrc.json]: why is "object-shorthand" included twice?
What exactly does that do?

File: .eslintrc.json (hunk 1 above)
Comment (devorah) [reply .eslintrc.json:13]: Well, kind of. We'll have to do an ES6 session at some point.

Btw, you can officially approve this PR by going to "Files Changed" tab. On the top right there is a "Review changes" button, and inside you can approve :)

**Review by tova-moskowitz — APPROVED** [review]

**Review by tova-moskowitz — APPROVED** [review]

File: .eslintrc.json (hunk 2 above)
Comment (devorah) [reply .eslintrc.json]: Good catch!!

Comment (devorah) [reply .eslintrc.json]: Object shorthand allows you to do the following:

```js
// es5
var foo = {
    doSomething: () {
        console.log('I did something!');
    }
};

// es6
var foo = {
    doSomething () {
        console.log('I did something!');
    }
};
```

http://eslint.org/docs/rules/object-shorthand

Comment (devorah) [reply .eslintrc.json]: There are other implications too, all listed in that link

**Comment by devorah** [comment]
Thank you for catching that @tova-moskowitz! I fixed it (left it as a warning vs off). Maybe will help with transitioning to using es6

**Pull request merged**